h5netcdf/core.py

```diff
@@ -764,8 +777,49 @@ def _resize_variables(self, dimension, size, recurse=True):
 
 class File(Group):
     def __init__(
-        self, path, mode=None, invalid_netcdf=False, phony_dims=None, **kwargs
+        self, path, mode=None, invalid_netcdf=False, phony_dims=None,
+        *, track_order=True, **kwargs
```

Comment:
Do we need the `*` after `phony_dims`?

Reply:
Not necessary, but gives you more flexibility in changing the API later since you can reorder keyword arguments, while you can't re-order the position of positional argiuments so easily.

It also guides people to write more readable code.

Reply:
Great.

Reply:
This has been reverted now that the keyword argument has been removed.